terraform/environments/core-vpc/iam.tf

```diff
@@ -1,3 +1,48 @@
 data "aws_iam_role" "vpc-flow-log" {
   name = "AWSVPCFlowLog"
 }
+
+resource "aws_iam_policy" "policy" {
+  name        = "s3_dynamo_access_policy"
+  path        = "/"
+  description = "S3 and dynamo policy"
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Action" : "s3:ListBucket",
+        "Resource" : "arn:aws:s3:::mybucket"
```

Comment:
Probably want to change this to refer to the correct bucket.
Have you considered maybe using the output of a data call to populate this, to keep you from needing a hard coded reference in the policy?

Reply:
Bucket name corrected.